Assets/Scripts/Combat/Projectile.cs

```diff
@@ -9,12 +9,32 @@ public class Projectile : MonoBehaviour
     Health target = null;
     float damage = 0;
 
+    bool guidedMissile = false;
+    bool projectileFired = false;
+    Vector3 savedLocation;
+
     // Update is called once per frame
     void Update()
     {
         if (target == null) return;
 
-        transform.LookAt(GetAimLocation());
+        if (!projectileFired && !guidedMissile) {
+            savedLocation = GetAimLocation();
+            projectileFired = true;
+
```

Comment:
extra line

Reply:
Yeah, there I liked curly brackets style more so else if underneath was looking odd so I added the line